README.md

```diff
@@ -10,22 +10,19 @@
 <!--END STABILITY BANNER-->
 
 This library provides a high level, extensible pattern for constructing services
-deployed using Amazon ECS.
+deployed using Amazon ECS. It is named `@aws-cdk-containers/ecs-service-extensions` for
+TypeScript and JavaScript and named `cdk-ecs-service-extensions` for other JSII target
+languages.
 
 ```ts nofixture
-import {
-  AppMeshExtension,
-  CloudwatchAgentExtension,
-  Container,
-  Environment,
-  FireLensExtension,
-  HttpLoadBalancerExtension,
-  Service,
-  ServiceDescription,
-  XRayExtension,
-} from 'cdk-ecs-service-extensions';
+import * as ecs_extensions from '@aws-cdk-containers/ecs-service-extensions';
```

Comment:
Moved it back. I added it as its own snippet because i tried to make it compliant with `rosetta extract` but i couldn't get it to work properly so I rolled it back. 

This import was originally just part of the first example snippet. I agree it should be somewhere in the readme, so I reverted this change.